BIDebugEngine/BIDebugEngine/Debugger.cpp

```diff
@@ -835,9 +835,10 @@ void Debugger::grabCurrentCode(JsonArchive& answer, const std::string& file) con
             return false;
         });
 
-    }               
-    answer.Serialize("code", Script::getScriptFromFirstLine(breakStateInfo.instruction->instruction->sdp));
-    answer.Serialize("fileName", breakStateInfo.instruction->instruction->sdp.sourcefile);
+    } else {
+        answer.Serialize("code", Script::getScriptFromFirstLine(breakStateInfo.instruction->instruction->sdp));
```

Comment:
The if checks for nullptr, but the else then just uses it even if it might be nullptr.
Have you experienced it ever being null?